server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -294,6 +295,9 @@ public final class IndexSettings {
     }, new RefreshIntervalValidator(), Property.Dynamic, Property.IndexScope, Property.ServerlessPublic);
 
     static class RefreshIntervalValidator implements Setting.Validator<TimeValue> {
+
+        public static final String STATELESS_ALLOW_INDEX_REFRESH_INTERVAL_OVERRIDE = "stateless.allow.index.refresh_interval.override";
```

Comment:
I think we typically prefix these with "es", so let us call this:

```suggestion
        static final String STATELESS_ALLOW_INDEX_REFRESH_INTERVAL_OVERRIDE = "es.stateless.allow.index.refresh_interval.override";
```

(also make it package private)

Reply:
OK